frontend/src/editor/text-page/text-page.component.scss

```diff
@@ -0,0 +1,10 @@
+@import "../../assets/main-style.scss";
+
+.container {
+  background: $main-font-color;
+  color: #000000;
+  height: 1000px;
```

Comment:
1k px? Jest pewny? Jesli ma byc na wysokosc okna to np 100vh czy 100%. 
Ale wysokości 1k px... Chyba, że tak ma być to okej

Reply:
Wartość wysokości strony musi być stała oraz brałem wartość podobną do tej, która widnieje w rozmiarze okna w przeglądarce pdf w mozilli.